packages/core/src/physics/enums/HingeJointFlag.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Flags specific to the Hinge Joint.
+ */
+export enum HingeJointFlag {
```

Comment:
Add @internal, and don't need to export in `index.html`

Reply:
done